build/version.go

```diff
@@ -62,6 +61,22 @@ func init() {
 				"alphabet", r))
 		}
 	}
+
+	// Get build information from the runtime.
+	if info, ok := debug.ReadBuildInfo(); ok {
+		for _, setting := range info.Settings {
+			switch setting.Key {
+			case "vcs.revision":
+				CommitHash = setting.Value
+
+			case "-tags":
+				RawTags = setting.Value
+
+			case "go":
```

Comment:
This isn't a key in the `Settings` struct, it's `GoVersion` of the build info directly.

Reply:
LoL, when I tested it i printed: "go: info.GoVersion" and the for for settings after that.